src/Sulu/Bundle/AdminBundle/Resources/js/containers/Login/ForgotPasswordForm.js

```diff
@@ -12,7 +12,7 @@ import type {ElementRef} from 'react';
 type Props = {|
     loading: boolean,
     onChangeForm: () => void,
-    onSubmit: (user: string) => void,
+    onSubmit: (data: Object) => void,
```

Comment:
think i would add types for these objects in the `types.js` 🙂

Reply:
could look like this:

```
export type ForgotPasswordData = {
    user: string,
};
```

Reply:
not sure, defining the type here would strict it to specific fields, but the point of the PR is to make it more flexible and the backend decide with which data it can work.

Reply:
if you define the type like in the example above, flow allows additional fields 🙂  

additional fields are not allowed only if you use the exact object type syntax like this:
```
export type Test = {| user: string |};
```

Reply:
looks like we could also explicitly state that the object type is inexact by adding a `...` to the type 😅 

```
export type ForgotPasswordData = { user: string, ...};
```

https://flow.org/en/docs/types/objects/#toc-explicit-inexact-object-types